gbpservice/nfp/service_vendor_agents/vyos/oc_config_server/visibility_api.py

```diff
@@ -0,0 +1,55 @@
+# One Convergence, Inc. CONFIDENTIAL
+# Copyright (c) 2012-2015, One Convergence, Inc., USA
+# All Rights Reserved.
+#
+# All information contained herein is, and remains the property of
+# One Convergence, Inc. and its suppliers, if any. The intellectual and
+# technical concepts contained herein are proprietary to One Convergence,
+# Inc. and its suppliers.
+#
+# Dissemination of this information or reproduction of this material is
+# strictly forbidden unless prior written permission is obtained from
+# One Convergence, Inc., USA
+
+import logging
+import subprocess
+
+from vyos_session import utils
+
+OP_SUCCESS = True
+OP_FAILED = False
+
+logger = logging.getLogger(__name__)
+utils.init_logger(logger)
+
+class APIHandler(object):
+    def __init__(self):
+        pass
+
+    def run_command(self, command):
+        proc = subprocess.Popen(command,
```

Comment:
I think, it is better to use check_output rather than Popen. Please check the differences.